### DISCUSSION TITLE
`@extern(module, "module_name")` implemented yet?

### DISCUSSION THREAD

**Discussion by rempas** [MrSmith33/vox#19] [General]
Is this feature implemented yet? If not, I'll leave this open so you can add the answer when it gets implemented so I know.

**Comment by MrSmith33** [comment]
Haven't touched it yet. Will write you when I do it.

**Comment by MrSmith33** [comment]
We now have `@extern(module, "module_name")` implemented for jit and for windows targets.

New hello world example: <https://gist.github.com/MrSmith33/34a7557ad5ac23ebe6cf27bef15a39a6>
